### PR TITLE
Add log_output configuration with constraints

### PR DESCRIPTION
**Pull request by vsharma6855**
JIRA:https://do-internal.atlassian.net/browse/DBAAS-5588

### CODE REVIEW

File: specification/resources/databases/models/advanced_config/mysql_advanced_config.yml [hunk 1]
```diff
@@ -254,3 +254,13 @@ properties:
     minimum: 1024
     maximum: 1048576
     example: 4096
+  log_output:
+    description: >-
+      Defines the destination for logs. Can be INSIGHTS, TABLE, or both (INSIGHTS,TABLE), or NONE to disable logs.
```
Comment (loosla) [inline specification/resources/databases/models/advanced_config/mysql_advanced_config.yml:259]: Looks like the order of items is important when we have multiple values - `INSIGHTS,TABLE` - as a "key": https://github.internal.digitalocean.com//digitalocean/cthulhu/blob/6cbc5a70bbef2702990933fabb6bbca9e14cfc2c/docode/src/do/teams/paas/dbaas/dbaas/aiven_provider.go#L6065-L6066 

Lets add some details about this case in a description:

```suggestion
      Defines the destination for logs. Can be `INSIGHTS`, `TABLE`, or both (`INSIGHTS,TABLE`), or `NONE` to disable logs. To specify both destinations, use `INSIGHTS,TABLE` (order matters).
```

File: specification/resources/databases/models/advanced_config/mysql_advanced_config.yml [hunk 2]
```diff
@@ -254,3 +254,13 @@ properties:
...
+    type: string
+    enum:
+      - INSIGHTS
+      - TABLE
+      - INSIGHTS,TABLE
+      - NONE
+    example: INSIGHTS
```
Comment (loosla) [inline specification/resources/databases/models/advanced_config/mysql_advanced_config.yml:266]: I may be mistaken, but it seems like we expect one of the enum values for log_output based on this reference:
[link](https://github.internal.digitalocean.com//digitalocean/cthulhu/blob/6cbc5a70bbef2702990933fabb6bbca9e14cfc2c/docode/src/do/teams/paas/dbaas/dbaas/aiven_provider.go#L6065-L6066).

Would it be helpful to:
- set a default value in code and in a documentation or 
- clarify in the documentation that this parameter is required or 
- even do both?

File: specification/resources/databases/models/advanced_config/mysql_advanced_config.yml (hunk 1 above)
Comment (loosla) [reply specification/resources/databases/models/advanced_config/mysql_advanced_config.yml:259]: @andrewsomething Do you happen to remember if we have any combined enum values like `INSIGHTS,TABLE` mentioned in the documentation? What's the best way to explain this clearly? 🙏

File: specification/resources/databases/models/advanced_config/mysql_advanced_config.yml (hunk 2 above)
Comment (vsharma6855) [reply specification/resources/databases/models/advanced_config/mysql_advanced_config.yml:266]: https://github.internal.digitalocean.com/digitalocean/cthulhu/blob/e04cc1b45af1f348e0063717838a5ffcd9cc46f3/docode/src/do/teams/paas/dbaas/dbaasapi/config/config.go#L569C1-L575C1
Above is the actual Configs,
We are already having default value as NONE

**Review by andrewsomething — DISMISSED** [review]
:+1:

File: specification/resources/databases/models/advanced_config/mysql_advanced_config.yml [hunk 3]
```diff
@@ -254,3 +254,13 @@ properties:
...
+  log_output:
+    description: >-
+      Defines the destination for logs. Can be `INSIGHTS`, `TABLE`, or both (`INSIGHTS,TABLE`), or `NONE` to disable logs. To specify both destinations, use `INSIGHTS,TABLE` (order matters). Default is NONE.
+    type: string
+    enum:
+      - INSIGHTS
+      - TABLE
+      - INSIGHTS,TABLE
+      - NONE
+    example: INSIGHTS
```
Comment (loosla) [inline specification/resources/databases/models/advanced_config/mysql_advanced_config.yml:266]:
```suggestion
    example: INSIGHTS
    default: NONE
```

**Review by loosla — DISMISSED** [review]
:+1: Left a minor suggestion.

**Review by loosla — APPROVED** [review]
:rocket:

**Pull request merged**